dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jakarta.concurrency-2.0.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=com.ibm.websphere.appserver.jakarta.concurrency-2.0
 visibility=private
 singleton=true
--bundles=com.ibm.websphere.jakarta.concurrency.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0-RC1"
+-bundles=io.openliberty.jakarta.concurrency.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0-RC1"
 kind=ga
```

Comment:
this should be kind=beta

Reply:
Actually it needs to remain GA. This is due to the current arrangement where a common codebase implements both Java EE/Jakarta at once.  If we were to remove this feature from GA, the common impl that is used for concurrent-1.0 would break.